Foundation/Bundle.swift

```diff
@@ -54,7 +54,26 @@ open class Bundle: NSObject {
     }
     
     public init(for aClass: AnyClass) {
-        NSUnimplemented()
+        let pointer = unsafeBitCast(aClass, to: UnsafeRawPointer.self)
```

Comment:
Just wondering, why are we using unsafeBitCast here and not an inout?

```
var theClass = aClass
if let path = _CFBundleBlah(&theClass) {...}
```

My understanding is they do the same thing but the latter is more resilient to changes in ABI